source/index.txt

```diff
@@ -32,145 +38,329 @@ Atlas App Services
    {+client-sdks+} <https://mongodb.com/docs/realm>
    Get Help </help>
 
-Atlas App Services provides a suite of managed cloud services including Atlas Device Sync,
-serverless cloud functions, declarative data access rules, and more. You can use
-App Services to write and host a full application in a managed cloud
-environment backed by MongoDB Atlas.
+.. introduction::
+   
+   Atlas Application Services are fully-managed backend services and
```

Comment:
> .. fully-managed backend services... help you build apps and services... use the services to build apps... other services...

Nit: there is a lot of "services" and "build apps" here. Could we wordsmith this a bit more? Maybe in a follow-up PR because I don't want to block this.

What about something like:

> .. fully-managed backend services and APIs that help you develop faster with MongoDB Atlas. Use App Services to build apps, integrate Atlas...